gh001-workflows/caller-workflows/pr-labeler.yaml

```diff
@@ -0,0 +1,17 @@
+name: PR Labeler
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
Is the workflow creating the labels?

Reply:
The workflow attaches pre-existing labels. The ones defined in the `prl-config.yaml` file